src/components/evaluation/EvaluationActions.vue

```diff
@@ -178,7 +192,9 @@ export default {
           this.selectedEvaluationIds,
           fields
         ).then(data => this.afterApply(action, data), error => this.showErrorDialog(error.response.data.message))
-        .finally(() => this.setDisableControls(false))
+        .finally(() => this.setDisableControls(false),
+          this.isLoading = false
+        )
```

Comment:
@cesarvh - the `isLoading = false` line above is nested in a conditional. Is there a change it won't be reached and `isLoading` will remain `true`?

Reply:
You're right. Moved it to inside the conditional so we know itll be executed for sure